MusicTaste/MusicTaste/View Controllers/DetailsViewController.m

```diff
@@ -14,24 +16,27 @@ - (void)viewDidLoad {
     [super viewDidLoad];
     //Set Username, Image, and Bio
     self.screenName.text = [self.author valueForKey:@"username"];
-    
-    
+    self.bio.text = [self.author valueForKey:@"bio"];
```

Comment:
Sets Bio, image, and name to Profile View using Parse and the author that we passed through